src/ASM/ASMutils.C

```diff
@@ -176,7 +176,7 @@ bool ASM::readXML (std::istream& is, IntMat& MNPC, std::vector<Vec3>& nodes,
   {
     Vec3 X;
     std::istringstream iss(data);
-    for (size_t inod = 0; iss; inod++)
+    for ([[maybe_unused]] size_t inod = 0; iss; inod++)
```

Comment:
This was a new thing I did not know about, but think I would prefer the (maybe old-fashioned):
```
#if SP_DEBUG > 1
  for (size_t inod = 0; iss; inod++)
#else
  while (iss)
#endif
```
However, I don't insist.

Reply:
okay.